### PR TITLE
add haproxy_status metric

### PR DESCRIPTION
**Pull request by bjsignalfx**

### CODE REVIEW

File: internal/monitors/haproxy/metadata.yaml [hunk 1]
```diff
@@ -59,6 +59,10 @@ monitors:
       unique_proxy_id:
         description: 'A unique proxy id'
     metrics:
+      haproxy_status:
+        default: false
+        description: '. Values reported for listeners, frontends, backends and servers.'
```
Comment (keitwb) [inline internal/monitors/haproxy/metadata.yaml]: Leading `.`.

Comment (keitwb) [reply internal/monitors/haproxy/metadata.yaml]: Also this description is quite vague.  What kind of values is this supposed to be?  Is it some kind of enum?

Comment (keitwb) [reply internal/monitors/haproxy/metadata.yaml]: Ok, better now but you need to remake the docs.

Comment (bjsignalfx) [reply internal/monitors/haproxy/metadata.yaml]: Remade the docs

**Pull request merged**